pkgs/tools/misc/bandwidth/default.nix

```diff
@@ -19,7 +19,8 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ nasm ];
 
-  buildFlags = [ arch ];
+  buildFlags = [ arch ]
+    ++ stdenv.lib.optionals stdenv.cc.isClang [ "CC=clang" "LD=clang" ];
```

Comment:
This comes back a lot, I wonder if stdenv shouldn't be defining CC and LD instead... @copumpkin?